recipes/topicmapping/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: topicmapping
+  version: 1.0.0
+
+source:
+  - url: https://bitbucket.org/andrealanci/topicmapping/get/13bb57316ff0.zip
```

Comment:
Is that a specific hash? Can we get it from a stable release?

Reply:
This is the only and stable release.

It is the code written for a research paper ( https://arxiv.org/abs/1402.0422 ) and it is not longer developed (it is possible that previous commits had been lost in a change of repository)

This package would be useful to manage dependencies for future research projects.

Reply:
They don't really have tags but it seems that the paper calls it "v1" so let's go with that.